helix-core/src/main/java/org/apache/helix/controller/rebalancer/topology/Topology.java

```diff
@@ -54,14 +55,7 @@
   private final List<String> _liveInstances;
   private final Map<String, InstanceConfig> _instanceConfigMap;
   private final ClusterConfig _clusterConfig;
```

Comment:
Can put all the related info to ClusterTopologyConfig and remove the reference of ClusterConfig here?

Reply:
Good point, updated. thx